packages/blade/src/components/Input/BaseInput/BaseInputVisuals.tsx

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
Have we kept place for `isLoading` spinner for when we need to add it here?

Reply:
interactionElement is that.